CustomFeeValidator.cs

```diff
@@ -51,12 +51,14 @@ static bool Load(UnityModManager.ModEntry modEntry)
         static void OnGui(UnityModManager.ModEntry modEntry)
         {
             loggingEnabled = GUILayout.Toggle(loggingEnabled, "enable logging");
+            GUILayout.Label("Fee Validation Type:");
             try
             {
                 selectedFeeType = (FeeType)Enum.Parse(typeof(FeeType), feeTypeNames.ElementAt(GUILayout.SelectionGrid(

```

Comment:
UMM supports direct drawing of Enums without needing to convert back and forth manually: https://wiki.nexusmods.com/index.php/How_to_render_mod_options_(UMM)

Reply:
I'll take a look at this, thanks! I'm totally new to mod settings, so still figuring things out. 😅

Reply:
This was the "secret" I needed to find! 😁 https://www.codingame.com/playgrounds/2487/c---how-to-display-friendly-names-for-enumerations